src/main/java/cz/startnet/utils/pgdiff/PgDiffViews.java

```diff
@@ -33,7 +33,15 @@ public static void createViews(final PrintWriter writer,
             final PgSchema oldSchema, final PgSchema newSchema,
             final SearchPathHelper searchPathHelper) {
         for (final PgView newView : newSchema.getViews()) {
-            final PgView oldView = oldSchema.getView(newView.getName());
+        	PgView oldView;
+        	if (oldSchema != null)
+        	{
+        		oldView = oldSchema.getView(newView.getName());
+        	}
+        	else
+        	{
+        		oldView = null;
+        	}
```

Comment:
You could do it a little more neatly I think.

```suggestion
        	PgView oldView = (oldSchema == null ? null : oldSchema.getView(newView.getName()));
```

Reply:
Ok, committed, thanks